src/Widgets/LanguageSwitchWidget/LanguageSwitchWidgetComponent.tsx

```diff
@@ -9,8 +9,10 @@ provideComponent(LanguageSwitchWidget, ({ widget }) => {
   if (alignment) classNames.push(alignment)
 
   return (
```

Comment:
Unrelated to this code: I'm missing a Thumbnail for the new widget. Also: Is the name "Language Switch" a good name for regular editors?